src/impact-dashboard/FacilityInformation.tsx

```diff
@@ -112,9 +126,22 @@ export const AgeGroupGrid: React.FC<AgeGroupGridProps> = ({
 }) => {
   const [collapsed, setCollapsed] = useState(collapsible);
 
+  const collapseAgeInputs = () => {
+    if (pastAgesKnown(props.model)) {
+      return false;
+    } else if (
+      pastAgesKnown(props.model) === false &&
+      pastAgesUnknown(props.model) === true
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
   useEffect(() => {
-    setCollapsed(collapseAgeInputs(props.model));
-  }, [props.model]);
+    setCollapsed(collapseAgeInputs());
+  }, []);
```

Comment:
I am guessing the intention here is to only have this run when the component is first mounted; that's fine but you should disable the lint rule on this line and leave a comment to that effect

Reply:
I removed the props.model as a dependency specifically because that was causing the age inputs to toggle unexpectedly. So I'm guessing it's best to just leave it out.

Reply:
yeah that seems reasonable to me